src/main/java/org/springframework/data/couchbase/transaction/ReactiveTransactionsWrapper.java

```diff
@@ -31,6 +31,7 @@ public Mono<TransactionResult> run(Function<ReactiveTransactionAttemptContext, M
 
 					// This reactive context is what tells Spring operations they're inside a transaction.
 					.contextWrite(reactiveContext -> {
+						// todo gp why is this taking a TransactionOptions?
```

Comment:
It's never used.  Once upon a time I thought it would make sense to create an AttemptContext in here if it did not exist.